src/launch/policy.c

```diff
@@ -211,12 +220,12 @@ static int policy_at_uidgid(CRBTree *tree, PolicyNode **nodep, uint32_t uidgid)
         return 0;
 }
 
-static int policy_at_uid(Policy *policy, PolicyNode **nodep, uint32_t uidgid) {
-        return policy_at_uidgid(&policy->uid_tree, nodep, uidgid);
+static int policy_at_uid(Policy *policy, PolicyNode **nodep, uint32_t uid_start, uint32_t uid_end) {
+        return policy_at_uidgid(&policy->uid_tree, nodep, uid_start, uid_end);
 }
 
-static int policy_at_gid(Policy *policy, PolicyNode **nodep, uint32_t uidgid) {
-        return policy_at_uidgid(&policy->uid_tree, nodep, uidgid);
+static int policy_at_gid(Policy *policy, PolicyNode **nodep, uint32_t gid) {
+        return policy_at_uidgid(&policy->uid_tree, nodep, gid, gid);
```

Comment:
This needs to be `gid_tree`, doesn't it? Was wrong before. We should fix this now and rebase this on top.

Reply:
Fixed and pushed.